pkg/sql/schema_changer.go

```diff
@@ -2003,13 +2003,13 @@ func (sc *SchemaChanger) maybeReverseMutations(ctx context.Context, causingError
 			// If the mutation is for validating a constraint that is being added,
 			// drop the constraint because validation has failed.
 			if constraint := m.AsConstraint(); constraint != nil && constraint.Adding() {
-				log.Warningf(ctx, "dropping constraint %+v", constraint.ConstraintToUpdateDesc())
+				log.Warningf(ctx, "dropping constraint %s", constraint.String())
```

Comment:
don't call `String`, as a habit it'll lead to immediate redaction

Reply:
Fixed.